src/shmem_env_defs.h

```diff
@@ -107,6 +107,8 @@ SHMEM_INTERNAL_ENV_DEF(OFI_STX_ALLOCATOR, string, "round-robin", SHMEM_INTERNAL_
                        "Algorithm for allocating STX resources to contexts")
 SHMEM_INTERNAL_ENV_DEF(OFI_STX_DISABLE_PRIVATE, bool, false, SHMEM_INTERNAL_ENV_CAT_TRANSPORT,
                        "Disallow private contexts from having exclusive STX access")
+SHMEM_INTERNAL_ENV_DEF(DISABLE_MULTIRAIL, bool, false, SHMEM_INTERNAL_ENV_CAT_TRANSPORT,
+                       "Disable usage of multirail functionality")
```

Comment:
Let's add this to the README file, noting that it only affects the OFI transport.

In fact, for now we should probably call it `OFI_DISABLE_MULTIRAIL`.

Reply:
Ok! If we change DISABLE_MULTIRAIL to OFI_DISABLE_MULTIRAIL is it necessary to explicitly mention in the README that it only affects the OFI transport?